packages/services/src/models/notification-scanners/DisputeRuled.js

```diff
@@ -0,0 +1,45 @@
+import NotificationScannerBaseModel from './NotificationScannerBaseModel'
+import Network from '@aragonone/court-backend-server/build/web3/Network'
+
+class DisputeRuled extends NotificationScannerBaseModel {
+  async scan() {
+    let notifications = []
+    const nineDaysBeforeNow = Math.floor((Date.now()-(9*this._DAYS))/1000)
+    const query = `
+    {
+      adjudicationRounds(where: {state: Ended, createdAt_gt: ${nineDaysBeforeNow}}, orderBy: createdAt) {
+        dispute {
+          id
+        }
+        jurors {
```

Comment:
Should all jurors get DisputeRuled notifications including missed vote/reveal?

Reply:
Same answer as above